website/content/docs/bloop-basics.md

```diff
@@ -51,52 +52,133 @@ Bloop is an agnostic build server that makes hot compiler accessible to all
+same problems a build tool does! The problems that bloop aims to address are:
 
-#### Command line friendly
+* Compiling, testing and running code in the fastest way possible.
+* Provide the backbone so that other tools can build upon it.
 
-The primary way to interact with bloop is directly from the command line
-instead of through a shell or repl. This workflow is more familiar to how
-most of developer tools work (for example, Maven, Gradle or sbt) and allows
-you to reuse the same terminal for other tasks.
+It can then be used by Scala developers to be more productive and by tooling
+authors to write their tools faster and more reliably. As a result, many
+build tools, IDEs and client can reuse Bloop to have a fast developer
+workflow (and avoid manually integrating with Scala and Java incremental
+compilers).
+
+#### A note on the future of build tools
+
+It's an exciting time for Scala tooling!
 
-#### Simplicity
+There is a lot of activity in the build tools field; many smart
+people are working on how to make build tools in the Scala ecosystem faster,
+easier to use and more correct.
 
-You can think of bloop as a simple, end-user application that runs Bloop's
-core commands. The does not require you to learn a complex DSL or
-configuration format to interact with it. `--help` takes you a long way.
+Among these efforts we find organizations like Stripe or Wix working on
+[better Scala support for Bazel](scala-bazel) or Twitter getting excellent
+Scala support in [Pants](pants), and individuals like [Jan Christopher
+Vogt](@cvogt) or [Li Haoyi](lihaoyi) working on [cbt] and [mill]
+respectively.
+
+All of these are exciting projects. In the future, we expect some of them to
+integrate with Bloop and benefit from a faster cross build-tool workflow! But
+we're also aware that they incidentally address the problem of compiling
+faster in different ways.
+
+With this in mind, we've created Bloop to primarly support all the existing
+build tools that companies and Scala open source developers use *today*. We
+believe that it's possible for many of these new tooling efforts to use Bloop
+to get a faster developer workflow, but that's not our focus since the
+community around them is small at the moment.
+
+By supporting reliable and hardwearing JVM build tools like sbt, Maven and
+Gradle, altogether with IDEs (Intellij) and the ongoing tooling experiments,
+we want to improve the life of every Scala developer.
+
+### Other bloop features
 
 #### Concurrent
 
 A single bloop server handles concurrent requests from multiple clients. This
 enables several clients to reuse the same bloop instance without blocking
 each other.
 
+#### Runtime independent (on the roadmap)
+
+Test or run your Scala application on the JVM, Scala.js and Scala Native.
+Bloop will with the API of every runtime that the Scala ecosystem supports so
+that external tools don't have to.
+
+[There is ongoing effort in the Scala Native build API](), contributed by
```

Comment:
The address is missing here: https://github.com/scala-native/scala-native/pull/1143